package.json

```diff
@@ -79,12 +79,6 @@
             "description": "Allows the user to configure if Snyk analysis is run automatically, manually or every 30 minutes. Default is automatically on save.",
             "scope": "application"
           },
-          "snyk.advanced.codeEnabled": {
```

Comment:
This removes the setting from UI, but not in settings.json, right?

Reply:
That's right, we don't have any migrations for it. It would be nice to have one if we further continue juggling with our settings. Not sure if it's worth introducing it now as this doesn't hurt our users. Wdyt?

Reply:
I don't think we should migrate it now (was more understanding for me).